interpreter/terminal_interface/validate_llm_settings.py

```diff
@@ -25,7 +25,9 @@ def validate_llm_settings(interpreter):
 
             # OpenAI
             if interpreter.llm.model in litellm.open_ai_chat_completion_models:
-                if not os.environ.get("OPENAI_API_KEY") and not interpreter.llm.api_key:
+                if (
+                    True
```

Comment:
Is this an artifact left over from testing?

Reply:
Yes xD